src/main/java/com/uid2/optout/partner/OptOutPartnerEndpoint.java

```diff
@@ -85,6 +94,8 @@ public Future<Void> send(OptOutEntry entry) {
                 }
 
                 if (SUCCESS_STATUS_CODES.contains(resp.statusCode())) {
+                    long finishTimeMs = System.currentTimeMillis();
+                    timer.record(finishTimeMs - startTimeMs, TimeUnit.MILLISECONDS);
```

Comment:
Currently only recording successful sends - failures could happen for a range of reasons that might affect the time they take, so having a timer on "any failed request" probably isn't useful.